pkg/systemd/services/service-details.jsx

```diff
@@ -452,6 +452,16 @@ export class ServiceDetails extends React.Component {
                                     <DescriptionListTerm>{ _("Path") }</DescriptionListTerm>
                                     <DescriptionListDescription id="path">{this.props.unit.FragmentPath}</DescriptionListDescription>
                                 </DescriptionListGroup>
+                                {this.props.unit.MemoryCurrent && <DescriptionListGroup>
```

Comment:
Is this how much it currently consumes? If so, is it dynamical changing over time?

Reply:
It is but it's also not updated asynchronously right now, let me see how I can get an event for that.

Very nice catch ;)

Reply:
So this seems a bit hard, according to systemd documentation this property never emits a signal when changed. https://man7.org/linux/man-pages/man5/org.freedesktop.systemd1.5.html

Reply:
Then I suppose it could update itself every 30s or so through a timer?

Reply:
Ok polling it is.